tests/api/test_get_resource.py

```diff
@@ -500,11 +500,6 @@ def test_get_resources(client, resource, uid, expected):
         'repo_name': 'some_repo_name',
         'repo_namespace': 'some_repo_namespace'
     })[0]
-    push = DistGitPush.get_or_create({
```

Comment:
Looks like this file still has references to this push object in relationship definition and expected output above. Could you please remove them and try running the tests locally by first doing a `systemctl start docker` and then `sudo ./scripts/run-tests.sh` to see if they all pass?